clang/test/dpct/syncthreads.cu

```diff
@@ -78,10 +81,12 @@ __global__ void test2(float *ptr1, float *ptr2) {
   for (;;) {
     ptr1[aaa];
     // CHECK:item_ct1.barrier(sycl::access::fence_space::local_space);
+    // CHECK:DPCT1112:{{.*}}: Synchronizing thread blocks inside a conditional branch statement may lead to undefined behavior
```

Comment:
Line84 is not necessary, in L81 foo, there is no condition at all.

Reply:
done